packages/jest-fake-timers/src/__tests__/legacyFakeTimers.test.ts

```diff
@@ -508,7 +508,7 @@ describe('FakeTimers', () => {
     });
 
     it('warns when trying to advance timers while real timers are used', () => {
-      const mockConsole = {warn: jest.fn()};
+      const mockConsole = {warn: jest.fn<(...args: Array<string>) => void>()};
```

Comment:
Just like all other? Or?

```suggestion
      const mockConsole = {warn: jest.fn<typeof console.warn>()};
```